_assets/ci/Jenkinsfile.tests

```diff
@@ -125,23 +130,10 @@ pipeline {
   } // post
 } // pipeline
 
-def getDefaultUnitTestCount() {
-  if (env.JOB_BASE_NAME == 'tests-nightly') {
-    return '20'
-  }
-  return '1'
-}
+def isTestNightlyJob() { env.JOB_BASE_NAME == 'tests-nightly' }
 
-def getDefaultUnitTestFailfast() {
-  if (env.JOB_BASE_NAME == 'tests-nightly') {
-    return false
-  }
-  return true
-}
+def getDefaultUnitTestCount() { isTestNightlyJob() ? '20' : '1' }
 
-def getDefaultTimeout() {
-  if (env.JOB_BASE_NAME == 'tests-nightly') {
-    return 5*60
-  }
-  return 40
-}
+def getDefaultUnitTestFailfast() { isTestNightlyJob() ? false : true }
```

Comment:
I wonder if you could just use `!isTestNightlyJob()` directly in `defaultValue`. If not then just here.

Reply:
Let me try.